hydroflow/src/util/mod.rs

```diff
@@ -229,36 +231,6 @@ where
     slice.sort_unstable_by(|a, b| f(a).cmp(f(b)))
 }
 
-#[cfg(test)]
```

Comment:
this is also in your other PR but i guess you want to check this one in first?

Reply:
Part of #942